### PR TITLE
chore(public-docsite-v9): enable build-less DX for storybook

### PR DESCRIPTION
**Pull request by Hotell**
## Previous Behavior

To be able to run `public-docsite-v9` one has to manually build all v8 and v0 packages. If any code is changed in v8 or v0 they again need to be re-build.

## New Behavior

`yarn workspace @fluentui/public-docsite-v9 storybook` works without any build steps needed. Also any change in v8,v0 code will be reflected without any rebuilds.

This drastically improves DX for existing and new contributors !

#### SWC
- swc is now used instead of babel for ts,tsx,js transpilation and instead of Terser for minification, via https://github.com/Karibash/storybook-addon-swc
- why the swc move:
  - speed 
  - v8 and v0 contain code that is[impossible to transpile via babel](https://github.com/microsoft/fluentui/issues/25767) or because non trivial storybook babel-loader configuration that explodes on v0 code that uses [constructor parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties) 
  - no need for [storybook babel overrides](https://github.com/microsoft/fluentui/blob/master/.storybook/main.js#L134) to avoid consuming per package babel config which causes issues with griffel babel preset

#### Path aliases merge

Under the hood we create tsconfig which contains all v0,v8,v9  ts congfigs merged path aliases. this config is then registered to `TsconfigPathAliasesPlugin`

## Related Issue(s)

* Follows https://github.com/microsoft/fluentui/pull/25778, https://github.com/microsoft/fluentui/pull/25854

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/62753a83dfe7c7bb522039cbfc59016d23464e31'>62753a83dfe7c7bb522039cbfc59016d23464e31</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/Avatar-mount_base.html">1292</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/Avatar-mount.html">1295</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/Button-mount_base.html">924</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/Button-mount.html">915</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProvider-mount_base.html">1504</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProvider-mount.html">1539</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProviderWithTheme-mount_base.html">575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProviderWithTheme-mount.html">583</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">548</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">538</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">568</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">592</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/InfoButton-mount_base.html">529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/InfoButton-mount.html">513</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/MakeStyles-mount_base.html">1956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/MakeStyles-mount.html">1957</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/Persona-mount_base.html">2832</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/Persona-mount.html">2811</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/SpinButton-mount_base.html">2361</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test-react-components/SpinButton-mount.html">2342</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/BaseButton-mount_base.html">1201</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/BaseButton-mount.html">1208</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Breadcrumb-mount_base.html">2819</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Breadcrumb-mount.html">2801</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Checkbox-mount_base.html">2626</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Checkbox-mount.html">2673</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/CheckboxBase-mount_base.html">2339</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/CheckboxBase-mount.html">2375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ChoiceGroup-mount_base.html">4325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ChoiceGroup-mount.html">4297</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ComboBox-mount_base.html">1196</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ComboBox-mount.html">1171</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/CommandBar-mount_base.html">9256</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/CommandBar-mount.html">9761</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ContextualMenu-mount_base.html">10099</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ContextualMenu-mount.html">10168</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DefaultButton-mount_base.html">1363</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DefaultButton-mount.html">1369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DetailsRow-mount_base.html">3252</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DetailsRow-mount.html">3252</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DetailsRowFast-mount_base.html">3264</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DetailsRowFast-mount.html">3263</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DetailsRowNoStyles-mount_base.html">3161</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DetailsRowNoStyles-mount.html">3119</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Dialog-mount_base.html">2949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Dialog-mount.html">3009</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DocumentCardTitle-mount_base.html">585</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/DocumentCardTitle-mount.html">572</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Dropdown-mount_base.html">3151</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Dropdown-mount.html">3165</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/FocusTrapZone-mount_base.html">1941</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/FocusTrapZone-mount.html">1987</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/FocusZone-mount_base.html">1943</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/FocusZone-mount.html">1940</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedList-mount_base.html">1782</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedList-mount.html">2037</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedList-virtual-rerender_base.html">1095</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedList-virtual-rerender.html">1082</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1575</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedList-virtual-rerender-with-unmount.html">1589</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedListV2-mount_base.html">565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedListV2-mount.html">577</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedListV2-virtual-rerender_base.html">563</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedListV2-virtual-rerender.html">558</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">563</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">557</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/IconButton-mount_base.html">1806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/IconButton-mount.html">1805</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Label-mount_base.html">746</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Label-mount.html">753</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Layer-mount_base.html">4178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Layer-mount.html">4206</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Link-mount_base.html">848</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Link-mount.html">853</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/MenuButton-mount_base.html">1596</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/MenuButton-mount.html">1615</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/MessageBar-mount_base.html">2340</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/MessageBar-mount.html">2337</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Nav-mount_base.html">3060</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Nav-mount.html">3067</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/OverflowSet-mount_base.html">1413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/OverflowSet-mount.html">1398</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Panel-mount_base.html">2586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Panel-mount.html">2483</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Persona-mount_base.html">1288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Persona-mount.html">1297</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Pivot-mount_base.html">1528</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Pivot-mount.html">1511</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/PrimaryButton-mount_base.html">1473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/PrimaryButton-mount.html">1488</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Rating-mount_base.html">6985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Rating-mount.html">6944</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SearchBox-mount_base.html">1482</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SearchBox-mount.html">1487</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Shimmer-mount_base.html">2908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Shimmer-mount.html">2918</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Slider-mount_base.html">2116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Slider-mount.html">2100</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SpinButton-mount_base.html">4323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SpinButton-mount.html">4315</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Spinner-mount_base.html">829</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Spinner-mount.html">826</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SplitButton-mount_base.html">2836</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SplitButton-mount.html">2862</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Stack-mount_base.html">860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Stack-mount.html">860</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/StackWithIntrinsicChildren-mount_base.html">2250</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/StackWithIntrinsicChildren-mount.html">2210</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/StackWithTextChildren-mount_base.html">4961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/StackWithTextChildren-mount.html">4900</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SwatchColorPicker-mount_base.html">9433</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/SwatchColorPicker-mount.html">9482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/TagPicker-mount_base.html">2349</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/TagPicker-mount.html">2372</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/TeachingBubble-mount_base.html">75008</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/TeachingBubble-mount.html">75434</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Text-mount_base.html">829</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Text-mount.html">819</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/TextField-mount_base.html">1566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/TextField-mount.html">1542</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ThemeProvider-mount_base.html">1433</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ThemeProvider-mount.html">1449</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ThemeProvider-virtual-rerender_base.html">1145</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ThemeProvider-virtual-rerender.html">1148</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1988</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1992</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Toggle-mount_base.html">1134</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/Toggle-mount.html">1136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/buttonNative-mount_base.html">535</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26476/perf-test/buttonNative-mount.html">540</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: .storybook/main.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 const path = require('path');
```
Comment (fabricteam) [inline .storybook/main.js:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26476&commitId=9e0525837110c5ec6455a5e24ca0cfcc78d62b9f&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**